bcda/bcdacli/cli.go

```diff
@@ -309,6 +314,26 @@ func setUpApp() *cli.App {
 		{
 			Name:     "generate-client-credentials",
 			Category: "Authentication tools",
+			Usage:    "Register a system and generate credentials for an ACO client specified by ACO CMS ID",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:        "cms-id",
+					Usage:       "CMS ID of ACO",
+					Destination: &acoCMSID,
+				},
+			},
+			Action: func(c *cli.Context) error {
+				msg, err := generateClientCredentials(acoCMSID)
+				if err != nil {
+					return err
+				}
+				fmt.Fprintln(app.Writer, msg)
+				return nil
+			},
+		},
+		{
+			Name:     "reset-client-credentials",
+			Category: "Authentication tools",
```

Comment:
Should we clarify in comments/description/usage that the client ID is preserved, and only the client secret is updated as part of this command?